index.js

```diff
@@ -0,0 +1,89 @@
+//********Objects
+
+const playerImg = document.querySelector('img')
+const playerName = document.querySelector('.player h2')
+const playerNickName = document.querySelector('.player em')
+const playerLikes = document.querySelector('.likes')
+const goalsUl = document.querySelector('#goals')
+const goalForm = document.querySelector('#new-goal-form')
+const likeBtn = document.querySelector('.like-button')
+
+
+//***********RENDER FUNCTIONS*************/
+
+const renderPlayer = player => {
+	playerImg.src = player.photo
+	playerName.textContent = player.name
+	playerName.id = player.id
+	playerNickName.textContent = player.nickname
+	playerLikes.textContent = `${player.likes} people have liked this player`
+	player.goals.forEach(goal => renderGoal(goal))
+}
+
+const renderGoal = goal => {
+	const goalLi = document.createElement('li')
+	goalLi.textContent = goal.description
+	goalsUl.append(goalLi)
+}
+
+
+/***********EVENT LISTENERS *************/
+
+goalForm.addEventListener('submit', logSubmit)
+likeBtn.addEventListener('click', addLike)
+
+function logSubmit(event) {
+	event.preventDefault()
+	const newGoal = {
+		playerId: parseInt(event.target.previousElementSibling.querySelector('h2').id),
```

Comment:
Here, it may have been easier to just add a dataset attribute with the player id directly to the form element instead of some other element, so you could just do `event.target.dataset.playerId`.